studio/components/grid/components/header/Header.tsx

```diff
@@ -114,7 +114,8 @@ const RowHeader: FC<RowHeaderProps> = ({ sorts, filters }) => {
   const onRowsDelete = () => {
     confirmAlert({
       title: 'Confirm to delete',
-      message: 'Are you sure you want to delete the selected rows? This action cannot be undone.',
+      message: `Are you sure you want to delete the selected ${selectedRows.size} rows? This action cannot be undone.`,
+      confirmText: `Delete ${selectedRows.size} rows`,
```

Comment:
Super nit but deleting 1 row should say "1 row" instead of "1 rows" like pagination: https://github.com/supabase/supabase/blob/master/studio/components/grid/components/footer/pagination/Pagination.tsx#L153